sdk/network/Microsoft.Azure.Management.Network/src/Microsoft.Azure.Management.Network.csproj

```
@@ -10,19 +10,13 @@
 			Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Network" (https://www.nuget.org/packages/Azure.ResourceManager.Network/). Please see the package changelog for more information.
 		</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>22.0.0</Version>
+		<Version>22.1.0.0-Beta.1</Version>
```

```suggestion
		<Version>22.1.0-Beta.1</Version>
```
You don't need 4 version digits here.

---

Fixed